shared/chat/conversation/info-panel/button-utils.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+import * as React from 'react'
+import {Box, Button, ButtonBar, Text} from '../../../common-adapters'
+import {globalMargins, globalStyles} from '../../../styles'
+
+const CaptionedButton = (props: {label: string, caption: string, onClick: () => void}) => (
```

Comment:
:sheep: This might be a good common adapter

Reply:
I agree, although I didn't look for other places this was used. If I see it again I'll see if I can pull it out.